### ISSUE TITLE
Fix example path of wiki2gfm.py in bulk-convert.sh

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Austin503/support-tools#41]
```
As seen in Revision ec307471f338
```

Original issue reported on code.google.com by `xslidian` on 16 Mar 2015 at 12:42

Attachments:
- [WikiToMarkdownTool.wiki.patch](https://storage.googleapis.com/google-code-attachments/support-tools/issue-41/comment-0/WikiToMarkdownTool.wiki.patch)

**Comment by GoogleCodeExporter** [comment]
```
Which file is this touching? I've fixed the wiki 
(https://code.google.com/p/support-tools/wiki/WikiToMarkdownTool) to reflect 
the new file path.
```

Original comment by `chrsm...@google.com` on 17 Mar 2015 at 7:08
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Yes, you've fixed exactly the same file in Revision 098f3fb78040 . Thanks!
```

Original comment by `xslidian` on 19 Mar 2015 at 6:40
